src/js/lib/monster.ui.js

```diff
@@ -3093,6 +3102,36 @@ define(function(require) {
 		}
 	};
 
+	/**
+	 * Get handlebars templateto render an SVG icon
+	 * @param   {Object} args
+	 * @param   {String} args.id                   Icon ID
+	 * @param   {String} [args.cssClass=svg-icon]  CSS classes to be applied to the SVG tag
+	 * @return  {String}                           SVG icon template
+	 */
+	function getTemplateSvgIcon(args) {
+		if (!_.isPlainObject(args)) {
+			throw TypeError('"args" is not a plain object');
+		}
+		if (!_.has(args, 'id') || _.isUndefined(args.id)) {
+			throw TypeError('"args.id" is undefined');
+		}
+		if (!_.isString(args.id)) {
+			throw TypeError('"args.id" is not a string');
+		}
+		if (_.has(args.cssClass) && !_.isString(args.cssClass)) {
+			throw TypeError('"args.cssClass" is not a string');
+		}
+		var iconId = args.id;
+		var iconPrefix = iconId.substring(0, iconId.indexOf('--'));
+		var templateData = {
+			iconId: iconId,
+			cssClass: _.get(args, 'cssClass', 'svg-icon ' + iconPrefix)
+		};
+
+		return monster.template(monster.apps.core, 'monster-svg-icon', templateData);
```

Comment:
To sence this to return a handlebars template: 
`return new Handlebars.SafeString(monster.template(monster.apps.core, 'monster-svg-icon', templateData));`

let me know if i'm wrong :)

Reply:
This function, on the other hand, is not a Handlebars helper, so it may be OK to return just the template as string, like `getTemplatePhoneNumber`.